aptoide-ui/src/main/java/cm/aptoide/pt/aptoide_ui/textformatter/TextFormatter.kt

```diff
@@ -48,6 +48,11 @@ class TextFormatter {
       return parseDate(unformattedDate, pattern).time
     }
 
+    fun formatDateToPattern(unformattedDate: String, pattern: String = "yyyy-MM-dd HH:mm:ss", newPattern: String): String {
```

Comment:
aptoide_ui is deprecated. maybe this should be moved to the extensions module

Reply:
Removed this function and used an alternative in the extensions module